docs/esa/iso/iso.rst

```diff
@@ -119,75 +115,86 @@ observation identifier (TDT) as input. There are different product levels:
 * HPDP: 'Highly Processed Data Products' (HPDP). These products include DATA (images, spectra etc.), which have been processed beyond the pipeline and/or using new, refined algorithms and therefore have been improved to any degree compared to the OLP 10 products, as well as any resulting CATALOGUES and ATLASES.
 * DEFAULT_DATA_SET: This is the selected pack of products that better characterize the observation, either pipeline products or HPDP products.
 
-There are two valid values for 'retrieval_type': OBSERVATION and STANDALONE. OBSERVATION is the one 
-used commonly to download data from the archive and STANDALONE returns a Virtual Observatory 
+There are two valid values for 'retrieval_type': OBSERVATION and STANDALONE. OBSERVATION is the one
+used commonly to download data from the archive and STANDALONE returns a Virtual Observatory
 product (if any).
 
-For more info on the data products, please check 'http://nida.esac.esa.int/nida-cl-web/', 
+For more info on the data products, please check 'http://nida.esac.esa.int/nida-cl-web/',
 "IDA USERS GUIDE" section.
 
-Both query and download methods are designed to be used in coordination. For example, we can query 
-for SWS observations with a certain target associated (not calibration) for the revolution 800 and 
+Both query and download methods are designed to be used in coordination. For example, we can query
+for SWS observations with a certain target associated (not calibration) for the revolution 800 and
 we can loop on the observations to download the best products (DEFAULT_DATA_SET) in different tar files.
 
-.. code-block:: python
+.. doctest-remote-data::
 
-  >>> from astroquery.esa.iso import ISO
-  >>> table=ISO.query_ida_tap(query="SELECT tdt, revno, aotname, ra, dec  FROM " + 
-  ...                               "ida.observations WHERE " + 
+  >>> from astroquery.esa.iso import ISO      # doctest: +IGNORE_OUTPUT
```

Comment:
why ignore the output again?

Reply:
I just missed it. All of these lines had ignores when I finished the rebase. 😛

Reply:
no worries, there might have been some INFO prints at the time of import, when this PR originally started.